rewrite-maven/src/test/java/org/openrewrite/maven/UpdateMavenProjectPropertyJavaVersionTest.java

```diff
@@ -249,6 +249,131 @@ void UpdateChildProperty() {
         );
     }
 
+    @Test
+    void doNotCrashOnImplicitVersion() {
+        rewriteRun(
+          mavenProject("spring-cloud-kubernetes",
+            pomXml(
+              """
+                <?xml version="1.0" encoding="UTF-8"?>
+                <!--
+                  ~ Copyright 2013-2020 the original author or authors.
+                  ~
+                  ~ Licensed under the Apache License, Version 2.0 (the "License");
+                  ~ you may not use this file except in compliance with the License.
+                  ~ You may obtain a copy of the License at
+                  ~
+                  ~      https://www.apache.org/licenses/LICENSE-2.0
+                  ~
+                  ~ Unless required by applicable law or agreed to in writing, software
+                  ~ distributed under the License is distributed on an "AS IS" BASIS,
+                  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+                  ~ See the License for the specific language governing permissions and
+                  ~ limitations under the License.
+                  ~
+                  -->
+                
+                <project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                         xmlns="http://maven.apache.org/POM/4.0.0"
+                         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+                
```

Comment:
```suggestion
                <project>
```

Reply:
I thought about removing these but thought we might get in trouble, then again, it's only a testcase